Assets/MixedRealityToolkit/Utilities/AssetScriptReferenceRetargeter.cs

```diff
@@ -343,10 +357,159 @@ private static bool IsYamlFile(string filePath)
             }
         }
 
-        private static void CleanupDirectory(string outputPath)
+        private static void PostProcess(string outputPath)
         {
             DirectoryInfo outputDirectory = new DirectoryInfo(outputPath);
             RecursiveFolderCleanup(outputDirectory);
+            CopyIntermediaryAssemblies(Application.dataPath.Replace("Assets", "NuGet/Plugins"));
+            UpdateMetaFiles();
+        }
+
+        private static void CopyIntermediaryAssemblies(string outputPath)
+        {
+            List<DirectoryInfo> outputDirectories = new List<DirectoryInfo>();
+            string playerDataCachePath = Application.dataPath.Replace("Assets", "Library/PlayerDataCache");
+            DirectoryInfo playerDataCahce = new DirectoryInfo(playerDataCachePath);
+            DirectoryInfo[] dllStores = playerDataCahce.GetDirectories("*", SearchOption.TopDirectoryOnly);
+            foreach(DirectoryInfo folder in dllStores)
+            {
+                string subfolderName = string.Empty;
+                switch(folder.Name)
+                {
+                    case "Win":
+                        subfolderName = "Standalone";
+                        break;
+                    case "WindowsStoreApps":
+                        subfolderName = "UAP";
+                        break;
+                    default:
+                        continue;
+                }
```

Comment:
We should be more explicit, perhaps declare a dictionary that maps between folder we expect to exist and it's output location. And then if the folder doesn't exist, we again thrown an error, as we can't create a complete package without it

Reply:
Sounds good - I'll get that in and submit the update with the rest of the changes